src/Client/style.scss

```diff
@@ -1,22 +1,15 @@
 @charset "utf-8";
 
-// FontAwesome
-$fa_font_path: "~@fortawesome/fontawesome-free/webfonts";
```

Comment:
I think we need to keep these, not sure though

Reply:
Stylesheets are now referenced in the index.html, i was sadly unable to get the project running with the references in styles.scss.

    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css">